src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
It's better to use the `for ... of` loop here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of